xcs-ui/xcs-ui-common/src/main/resources/FlamingoThemes/XCS-Blue.xml

```diff
@@ -946,9 +946,8 @@
 @gadget-shadow:none;
 /* livetable */
 @th-livetable-border:2px solid #999;
-@th-livetable-bg:#FFFFFF;
 @td-livetable-border:1px solid #ccc;
-@td-livetable-padd-btm:10px;
+@td-livetable-padd:5px 5px 5px 0;
```

Comment:
this padding is applied only for the filter td and not for the text, i haven't change the left padding value , the default value of xwiki is  padding: 2px 5px 2px 0; i have just keep the default value